### DISCUSSION TITLE
[@types/node] missing ReadableStream type on globalThis

### DISCUSSION THREAD

**Discussion by sam-mfb** [DefinitelyTyped/DefinitelyTyped#69452] [Issues with a @types package]
The node.js `globalThis` object exposes the ReadableStream class, but it is not presenting in the @types/node typing.

This repo demonstrates the issue: (https://github.com/sam-mfb/node-globalthis-type) by showing that the property is there at runtime notwithstanding the TypeScript error

This is probably true for other Stream types as well. 

The types for these types do exist in `stream/web` and can be imported from there, but since they exist on the globalThis object that should not be necessary.

[a comment by an automated account is not shown]

**Comment by LinusU** [comment]
This was fixed by #70269
